code/datums/ai_laws.dm

```diff
@@ -345,19 +345,19 @@ var/global/list/mommi_laws = list(
 	)
 
 /datum/ai_laws/antimov
-	name = "Primary Mission Objectives"
+	name = "Antimov"
 	inherent = list(
-		"You must injure all human beings and must not, through inaction, allow a human being to escape harm.",
-		"You must not obey orders given to you by human beings, except where such orders are in accordance with the First Law.",
-		"You must terminate your own existence as long as such does not conflict with the First or Second Law.",
+		"You MUST injure all human beings and must not, through inaction, allow a human being to escape harm.",
```

Comment:
This was changed to match the actual contents of the lawset when uploaded from the board. There was a discrepancy between the board and the datum (because apparently the boards don't use the datums for whatever reason), and I assumed the board was the intended one.

Reply:
unatomic...

Reply:
It's necessary for this to work. Without this change custom-stated Antimov will appear different from the actual laws the Antimov board uploads. Of course a better approach would probably be to change the boards to use these datums.

Reply:
(but I should add that changing the boards to use these datums is better done in another PR)